pkg/cli/admin/prune/imageprune/prune.go

```diff
@@ -18,7 +18,7 @@ import (
 
 	kappsv1 "k8s.io/api/apps/v1"
 	batchv1 "k8s.io/api/batch/v1"
-	batchv1beta1 "k8s.io/api/batch/v1beta1"
+
```

Comment:
This empty line will confuse people, there is no reason to split k8s.io imports into two groups.